sdk/secp256k1-program/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "solana-secp256k1-program"
+description = "Instructions for the Solana Secp256k1 native program."
+documentation = "https://docs.rs/solana-secp256k1-program"
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+bincode = { workspace = true, optional = true }
+digest = { workspace = true }
+libsecp256k1 = { workspace = true, features = ["hmac"] }
+serde = { workspace = true, optional = true }
+serde_derive = { workspace = true, optional = true }
+sha3 = { workspace = true }
+solana-feature-set = { workspace = true }
+solana-instruction = { workspace = true, features = ["std"] }
```

Comment:
Same here, we could make this dependency optional and include it when `"bincode"` is enabled.